data-8/jupyterhub-k8s
# Add user jovyan and give permissions over conda directory

Summary
-------
This diff adds user jovyan and gives him ownership of opt/conda so that the user can install conda packages in a notebook.

Additionally, we launch the notebook as jovyan and make the working directory /home/jovyan/work

How to test
-------
1. Launch a notebook and ensure that it works.
2. Launch a terminal and verify that we are logged in as user jovyan
